utils/release/github-release-changelog.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+# Access token will be required. Check to ensure that is it provied
+if [[ -z "$GITHUB_TOKEN" ]]; then
+	echo "GITHUB_TOKEN most be set"
+fi
+auth_header="-H Authorization: Bearer $GITHUB_TOKEN"
+
+script_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+source $script_dir/shared.sh
+
+# Get latest release version
+log "Getti previous release version tag"
```

Comment:
What about `latest` instead of previous which is more accurate?
```suggestion
log "Getting latest release version tag"
```

Reply:
I was going to use the word latest here, but I also felt it might be a bit misleading as the script is running to generate the new latest release.

Reply:
your call. `latest` expresses more accurately what the script is doing.